spec/tapioca/dsl/compilers/active_record_columns_spec.rb

```diff
@@ -182,24 +182,24 @@ def restore_id_value!; end
                       sig { returns(T.nilable([::Integer, ::Integer])) }
                       def saved_change_to_id; end
 
-                      sig { returns(T::Boolean) }
-                      def saved_change_to_id?; end
+                      sig { params(from: ::Integer, to: ::Integer).returns(T::Boolean) }
```

Comment:
Shouldn't we be changing these expectations to use `T.untyped` as well?

Reply:
Yep! Was in the process of pushing those changes up in e1197875112762b4100db126182d8978d2d1f72d and c29aa9ba028db33b54fd4f1bc95ec2ff945422b8.